src/resources/elements/sparkChart/spark-chart.ts

```diff
@@ -1,19 +1,29 @@
 import { AureliaHelperService } from "services/AureliaHelperService";
 import { autoinject } from "aurelia-framework";
 import "./spark-chart.scss";
-import { createChart, CrosshairMode, IChartApi, ISeriesApi } from "lightweight-charts";
+import { createChart, CrosshairMode, IChartApi, ISeriesApi, LineWidth, LineStyle} from "lightweight-charts";
 import { bindable } from "aurelia-typed-observable-plugin";
 import { NumberService } from "services/NumberService";
 
+interface ISeries {
+  name: string,
+  data: Array<any>,
+  color: string,
+  lineStyle?: LineStyle,
+  lineWidth?: LineWidth,
+}
+
 @autoinject
 export class SparkChart {
-  @bindable data: Array<any>;
+  @bindable data: Array<ISeries>;
+  @bindable.booleanAttr gridHorizontal = false;
+  @bindable.booleanAttr gridVertical = false;
   @bindable.booleanAttr interactive;
   @bindable.number height = 300;
   @bindable.number width = 500;
 
   chart: IChartApi;
-  series: ISeriesApi<"Line">;
+  series: Array<ISeriesApi<"Line">> = [];
```

Comment:
I'm using the `Line` type for the series, rather than the `Area`. In the `area` type, there's this added coloring below the line which is unnecessary according to the mocks.